### PR TITLE
Anime command using kitsu api + date refactoring utils

### PR DESCRIPTION
**Pull request by Etu34041**
Added : 

- anime command using kitsu api and fetching anime information corresponding to the name wrote by the user
- date refactoring utils (from 'yyyy/MM/dd' string or  'yyyy/MM/dd' string  to 'dd/MM/yyyy' string)

### CODE REVIEW

File: src/commands/anime/anime.js [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+        name: 'Start Date :calendar_spiral:',
+        value: anime.attributes.startDate != null ? refactorDate(anime.attributes.startDate) : 'Unknown',
+        inline: anime.attributes.endDate != null
+    });
+
+    if(anime.attributes.endDate != null) {
+        infosFields.push({
+            name: 'End Date :calendar_spiral:',
+            value: refactorDate(anime.attributes.endDate),
+            inline: anime.attributes.endDate != null
```
Comment (Becold) [inline src/commands/anime/anime.js]: `inline: true`

File: src/methods/refactorDateWithTime.js [hunk 2]
```diff
@@ -0,0 +1,6 @@
+module.exports = (date) => {
+    const dateAsArray = date.split('-');
+    console.log(dateAsArray);
+    console.log(dateAsArray[2]);
```
Comment (Becold) [inline src/methods/refactorDateWithTime.js]: Remove these 2 console.log

File: src/commands/anime/anime.js [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+            name: 'Episode Count',
+            value: anime.attributes.episodeCount,
+            inline: true
+        });
+    }
+
+    if(anime.attributes.youtubeVideoId != null){
+        infosFields.push({
+            name: 'Youtube video :arrow_forward:',
+            value: `${youtubeUrl}${anime.attributes.youtubeVideoId}`
```
Comment (Becold) [inline src/commands/anime/anime.js]: Can you create a link with markdown over here ?
```js
value: `[Click here](https://www.youtube.com/watch?v=${anime.attributes.youtubeVideoId})`
```

File: src/commands/anime/anime.js [hunk 4]
```diff
@@ -0,0 +1,94 @@
+const KitsuAPI = require('../../services/kitsu-api');
+const refactorDate = require('../../methods/refactorDate');
+const refactorDateWithTime = require('../../methods/refactorDateWithTime');
+const strUcFirst = require('../../methods/strUcFirst');
+function getAnimeInfos(anime) {
+    const infosFields = [];
```
Comment (Becold) [inline src/commands/anime/anime.js]: Please, use a `let` variable here.

File: src/commands/anime/anime.js [hunk 5]
```diff
@@ -0,0 +1,94 @@
...
+        name: 'Rating :star:',
+        value: `${anime.attributes.averageRating}/100`
+    })
+
+    return infosFields;
+}
+
+exports.exec = (bot, msg, args) => {
+    const Kitsu = new KitsuAPI();
+    let anime = [];
```
Comment (Becold) [inline src/commands/anime/anime.js]: Please, declare this variable as a `const` variable at line 67.

**Review by Becold — CHANGES_REQUESTED** [review]
Hi @MaximeLetecheur , ✋ 

Thank you for your contribution and opening this pull request.
I have reviewed the changes. Can you fix them, please ? 🙂

Thank you!
Romain

File: src/commands/anime/anime.js (hunk 1 above)
Comment (Etu34041) [reply src/commands/anime/anime.js]: my bad
correction done

**Comment by Etu34041** [comment]
Changes fixed in the next push

**Review by Becold — APPROVED** [review]

**Comment by Becold** [comment]
Thank you !

**Pull request merged**